auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -76,11 +79,47 @@ class CheckPullRequest extends RequestHandler {
     return filesIsEmpty;
   }
 
-  Future<bool> _removeLabel() async {
-    // TODO(Kristin): Implement the logic to remove the label. https://github.com/flutter/flutter/issues/99877
+  /// Removes the 'autosubmit' label if this PR should not be merged.
+  ///
+  /// Returns true if we successfully remove the label.
+  Future<bool> _removeLabel(
```

Comment:
It is more useful to send a comment to the PR with these log.info. Whenever we need to remove the label, the bot should leave a comment with corresponding info. Check the old logic.

Reply:
Thank you Keyong, I also think it's better to send a comment to PR. I will add this logic .